cmd/history.go

```diff
@@ -12,13 +13,15 @@ import (
 
 var (
 	historyCmd = &cobra.Command{
-		Use:     "history [IMAGE]",
-		Aliases: []string{"hist"},
-		Short:   "Clean up excessive image tags",
-		Long:    `Clean up excessive image tags matching the commit hashes (prefix) of the git repository`,
-		Args:    cobra.MinimumNArgs(1),
+		Use:          "history [PROJECT/IMAGE]",
+		Aliases:      []string{"hist"},
+		Short:        "Clean up excessive image tags",
+		Long:         `Clean up excessive image tags matching the commit hashes (prefix) of the git repository`,
+		Args:         cobra.MaximumNArgs(1),
+		SilenceUsage: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if err := validateHistoryCommandInput(args); err != nil {
+				cmd.Usage()
 				return err
 			}
```

Comment:
What's the point of silencing usage, but display it in case of errors either way?

Reply:
Here I just display it in case of a validation error. Other errors won't display it...